### PR TITLE
My first tasks

### PR DESCRIPTION
**Pull request by GRAFOLAF**
could you view my home task
THX a  lot =)

### CODE REVIEW

File: Nmaksimchicov/1.HTML/3.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<pre>print len(months)</pre>
+<p>которая вернёт 12.</p>
+<p>Ещё один пример списков — категории в поваренной книге. Например…</p>
+<pre>categories = ['Main dish','Meat','Fish','Soup','Cookies']</pre>
+<p>Таким образом, в <strong>categories[0]</strong> хранится <em>'Main dish'</em>, а в <strong>categories[4]</strong> — <em>'Cookies'</em>. Всё очень просто. Я уверен, вы сможете придумать много других примеров использования списков.</p>
+<hr width="100%" color="##000000" align="left" size="1">
+<h2>Комментарии</h2>
+<p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам. Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ, комментарии станут важными. Также при их помощи вы заставите Python игнорировать какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'. Например:</p>
+<p>#</p>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/3.html]: It is code

File: Nmaksimchicov/1.HTML/доп.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    	<h1><strong>Заголовок первого уровня</strong></h1>
+<h2><strong>Предварительно форматированный текст:</strong></h2>
+<pre>x | o | 
+‐‐‐‐‐‐‐‐‐   
+  | х |  
+‐‐‐‐‐‐‐‐‐‐ 
+o | x | o </pre>
+<hr width="100%" color="##000000" align="left" size="1">
+<h2><strong>Тег address</strong></h2>
+<em>Kuprevicha, 1/1, 220141, Minsk, Belarus. Phone: +375.17.2686911</em>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/доп.html]: should be tag address

File: Nmaksimchicov/1.HTML/доп.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+<hr width="100%" color="##000000" align="left" size="1">
+<h2><strong>Семантические и функциональные строчные теги</strong></h2>
+<p> Пожелания автора:</p>
+<p><del><font color="red"> Данный текст убрать из монографии</font></del> и заменить его на <a href="text.html"> вновь помещённый текст.</a></p>
+<hr width="100%" color="##000000" align="left" size="1">
+<h2><strong>Работа со строчными и блоковыми цитатами</strong></h2>
+<p>Цитата короткая:</p>
+<p><cite>И всё­таки она вертится!</cite></p>
+<p>Цитата занимающая несколько предложений:</p>
+ 		<p>О сколько нам открытий чудных</p>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/доп.html]: blockquote here

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Nmaksimchicov/1.HTML/3.html (hunk 1 above)
Comment (GRAFOLAF) [reply Nmaksimchicov/1.HTML/3.html]: could you be more specific.

Comment (pluhin) [reply Nmaksimchicov/1.HTML/3.html]: Why do you use paragraph tag for code section?

**Comment by GRAFOLAF** [comment]
What wrong ?

File: Nmaksimchicov/1.HTML/2.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    </HEAD>
+<!--начало документа -->
+    <BODY>
+        <h1><STRONG>HOW-TO: Программа на Python, Часть 1</STRONG></h1>
+        <ul>
+        <li><em>Программа на Python — часть 1</em></li>
+  		<li><em>Программа на Python — часть 2</em></li>
+ 		<li><em>Программа на Python — часть 3</em></li>
+		</ul>
+	<h2><STROMG>Содержание</STROMG></h2>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/2.html]: could you please remove strong, tag h2 does content bold

File: Nmaksimchicov/1.HTML/2.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Phython 01</TITLE>
+    </HEAD>
+<!--начало документа -->
+    <BODY>
+        <h1><STRONG>HOW-TO: Программа на Python, Часть 1</STRONG></h1>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/2.html]: could you please remove strong, tag h2 does content bold

File: Nmaksimchicov/1.HTML/2.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+		<li><ins>Наша первая программа</ins></li>
+		<li><ins> Пример цикла For</ins></li>
+	</ul>
+	<p>Из множества существующих языков программирования Python — один из самых лёгких
+	в изучении. Он создан в конце 80-х годов XX века, но с тех пор он сильно изменился.
+	Хотя Python входит в состав большинства дистрибутивов Linux, на него часто не
+	обращают внимания при выборе языка для изучения. В этой статье мы рассмотрим
+	создание консольных приложений, а в следующей — программирование графических
+	интерфейсов (<ins>GUI</ins>). Давайте создадим простую программу прямо сейчас.</p>
+	<h2><strong>Наша первая программа</strong></h2>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/2.html]: could you please remove strong, tag h2 does content bold

File: Nmaksimchicov/1.HTML/3.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Phython 02</TITLE>
+    </HEAD>
+<!--начало документа -->
+    <BODY>
+	<h1><strong>HOW-TO: Программа на Python, Часть 2</strong></h1>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/3.html]: could you please remove strong, tag h2 does content bold

File: Nmaksimchicov/1.HTML/3.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+<ul>
+	<li><em>Программа на Python — часть 1 </em></li>
+	<li><em>Программа на Python — часть 2</em></li>
+	<li><em>Программа на Python — часть 3</em></li>
+</ul>
+<h2><strong>Содержание</strong></h2>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/3.html]: could you please remove strong, tag h2 does content bold

File: Nmaksimchicov/1.HTML/3.html [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+<h2><strong>HOW-TO: Программа на Python, Часть 2</strong></h2>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/3.html]: could you please remove strong, tag h2 does content bold

File: Nmaksimchicov/1.HTML/3.html [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+<pre>print len(months)</pre>
+<p>которая вернёт 12.</p>
+<p>Ещё один пример списков — категории в поваренной книге. Например…</p>
+<pre>categories = ['Main dish','Meat','Fish','Soup','Cookies']</pre>
+<p>Таким образом, в <strong>categories[0]</strong> хранится <em>'Main dish'</em>, а в <strong>categories[4]</strong> — <em>'Cookies'</em>. Всё очень просто. Я уверен, вы сможете придумать много других примеров использования списков.</p>
+<hr width="100%" color="##000000" align="left" size="1">
+<h2>Комментарии</h2>
+<p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам. Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ, комментарии станут важными. Также при их помощи вы заставите Python игнорировать какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'. Например:</p>
+<code>#</code>>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/3.html:36]: extra >

File: Nmaksimchicov/1.HTML/4.html [hunk 11]
```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Пример страницы</TITLE>
+    </HEAD>
+<!--начало документа -->
+    <BODY>
+    	<h1><strong>HOW-TO: Программа на Python, Часть 3</strong></h1>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/4.html]: remove strong

File: Nmaksimchicov/1.HTML/4.html [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+<ul>
+<li><em>Программа на Python — часть 1</em></li>
+<li><em>Программа на Python — часть 2</em></li>
+<li><em>Программа на Python — часть 3</em></li>
+</ul>
+<h2><strong>Содержание</strong></h2>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/4.html]: remove strong

File: Nmaksimchicov/1.HTML/4.html [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+<h3><stromg>HOW-TO: Программа на Python, Часть 3</stromg></h3>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/4.html]: remove strong

File: Nmaksimchicov/1.HTML/4.html [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+import random
+# print 14 random integers
+for cntr in range(1,15):
+	print random.randint(1,10)</pre>
+<p>Давайте разберём код по строкам. Первые четыре строки — это комментарии, о которых
+мы говорили в прошлый раз. Пятая строка говорит Python, что нужно использовать
+модуль «random». Мы должны явно указать Python на это.</p>
+<hr width="100%" color="##000000" align="left" size="1">
+
+<h2><strong>Функции</strong></h2>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/4.html]: remove strong

File: Nmaksimchicov/1.HTML/5.html [hunk 15]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Пример страницы</TITLE>
+    </HEAD>
+<!--начало документа -->
+    <BODY>
+    	<h1><strong>Заголовок первого уровня</strong></h1>
+<h2><strong>Предварительно форматированный текст:</strong></h2>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/5.html]: remove strong

File: Nmaksimchicov/1.HTML/5.html [hunk 16]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Пример страницы</TITLE>
+    </HEAD>
+<!--начало документа -->
+    <BODY>
+    	<h1><strong>Заголовок первого уровня</strong></h1>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/5.html]: remove strong

File: Nmaksimchicov/1.HTML/5.html [hunk 17]
```diff
@@ -0,0 +1,71 @@
...
+<h2><strong>Предварительно форматированный текст:</strong></h2>
+<pre>x | o | 
+‐‐‐‐‐‐‐‐‐   
+  | х |  
+‐‐‐‐‐‐‐‐‐‐ 
+o | x | o </pre>
+<hr width="100%" color="##000000" align="left" size="1">
+<h2><strong>Тег address</strong></h2>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/5.html]: remove strong

File: Nmaksimchicov/1.HTML/5.html [hunk 18]
```diff
@@ -0,0 +1,71 @@
...
+    <blockquote>
+<p>Цитата занимающая несколько предложений:</p>
+        <p>О сколько нам открытий чудных</p>
+        <p>Готовят просвещенья дух</p>
+        <p>И опыт, сын ошибок трудных,</p>
+        <p>И гений, парадоксов друг,</p>
+        <p>И случай, бог изобретатель...</p>
+    </blockquote>    
+ <hr width="100%" color="##000000" align="left" size="1">
+<h2><strong>Семантические и функциональные строчные теги</strong></h2>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/5.html]: remove strong

File: Nmaksimchicov/1.HTML/доп.html [hunk 19]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```
Comment (pluhin) [inline Nmaksimchicov/1.HTML/доп.html]: Delicate file

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Nmaksimchicov/1.HTML/доп.html (hunk 19 above)
Comment (GRAFOLAF) [reply Nmaksimchicov/1.HTML/доп.html]: could you be more specific pls.

Comment (pluhin) [reply Nmaksimchicov/1.HTML/доп.html]: doublicate file. you have two files with additional taks

**Pull request merged**